Controls/Cafe/Cafe.h

```diff
@@ -8,6 +8,9 @@
  * your cafe challenge in Tests/Test_Cafe.c
  */
 
+// Function for checking error and printing if not NONE
+void checkError(OS_ERR err);
```

Comment:
Function naming could be a little more specific here, since it specifically handles one case

```C
void assertOSError(OS_ERR err);
```